Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM registry.opensuse.org/opensuse/leap:latest
+
+ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1 UWSGI_WSGI_FILE=/pcw/webui/wsgi.py UWSGI_MASTER=1
+ENV UWSGI_HTTP_AUTO_CHUNKED=1 UWSGI_HTTP_KEEPALIVE=1 UWSGI_LAZY_APPS=1 UWSGI_WSGI_ENV_BEHAVIOR=holy
+
+## System preparation steps ################################################# ##
+
+# Install system requirements
+RUN zypper -n in python3 python3-devel python3-pip gcc
+## Install SUSE CA. Comment the following two lines, if this is not desired
+RUN zypper -n ar http://download.suse.de/ibs/SUSE:/CA/openSUSE_Leap_15.3/ SUSE_CA
+RUN zypper -n in ca-certificates-suse
+COPY requirements.txt /
+# Install pip requirements
+RUN pip install -r requirements.txt
+# The cache (especially zypper) is not required for the resulting container
+RUN rm -rf /var/cache /requirements.txt
```

Comment:
This is useless and should be done after each `zypper in` call directly (as the same `RUN`